.travis/maybe-publish-coverage.sh

```diff
@@ -4,19 +4,19 @@ source $(npm root -g)/pr-bumper/.travis/is-bump-commit.sh
 
 if isBumpCommit
 then
-  echo "Skipping coverage publish for version bump commit"
+  echo "Skipping pr-bumper for version bump commit"
   exit 0
 fi
 
-if [ "$TRAVIS_NODE_VERSION" != "8.1.2" ]
+if [[ ! "${PUBLISH_NODE_VERSION:-8.1.2}" =~ ^$TRAVIS_NODE_VERSION ]]
 then
-  echo "Skipping coverage publish for TRAVIS_NODE_VERSION ${TRAVIS_NODE_VERSION}"
+  echo "Skipping pr-bumper for TRAVIS_NODE_VERSION [${TRAVIS_NODE_VERSION}]"
   exit 0
 fi
 
 if [ "$EMBER_TRY_SCENARIO" != "ember-default" ]
 then
-  echo "Skipping coverage publish for EMBER_TRY_SCENARIO ${EMBER_TRY_SCENARIO}"
+  echo "Skipping pr-bumper for EMBER_TRY_SCENARIO [${EMBER_TRY_SCENARIO}]"
```

Comment:
https://github.com/ciena-frost/ember-frost-core/pull/546/files#diff-872972d91442cb5b89f7fd8bebcf8a3eR19

Reply:
I know - https://github.com/ciena-frost/ember-frost-core/pull/547 changes it there as well